tests/integration.rs

```diff
@@ -345,3 +367,102 @@ fn sparse3(#[case] harness: Harness) {
 	let expected = [b'x'; 32768];
 	assert_eq!(buf, expected);
 }
+
+#[apply(all_images)]
+fn listxattr(#[case] harness: Harness) {
+	let d = &harness.d;
+
+	let file = File::open(d.path().join("xattrs")).unwrap();
+	let xattrs = file.list_xattr().unwrap().collect::<Vec<_>>();
+	let expected = [OsStr::new("user.test")];
+	assert_eq!(xattrs, expected);
+
+	freebsd! {
+		let num = unsafe { libc::extattr_list_fd(file.as_raw_fd(), EXTATTR_NAMESPACE_USER, std::ptr::null_mut(), 0) };
+		assert_eq!(num, 5); // strlen("test\0")
+	}
+}
+
+#[apply(all_images)]
+fn getxattr(#[case] harness: Harness) {
+	let d = &harness.d;
+
+	let file = File::open(d.path().join("xattrs")).unwrap();
+	let data = file.get_xattr("user.test").unwrap().unwrap();
+	let expected = b"testvalue";
+	assert_eq!(data, expected);
+
+	freebsd! {
+		// Can't use c"test" syntax, because the apply macro doesn't like it
+		let name = cstr!(b"test");
```

Comment:
"doesn't like it" how?  It works for me.
```suggestion
		let name = c"test";
```

Reply:
I now remember why, it's because `c""` literals were stabilized in 1.77.0,
while my PowerMac's rustc version is 1.74.1 for some reason,
even though it's FreeBSD 14.1,
which has rustc 1.79.0 on amd64.

Reply:
The original reason:
```
error: custom attribute panicked
   --> tests/integration.rs:172:1
    |
172 | #[rstest]
    | ^^^^^^^^^
...
427 | #[apply(all_images)]
    | -------------------- in this procedural macro expansion
    |
    = help: message: Unrecognized literal: `c"test"`
    = note: this error originates in the macro `all_images` which comes from the expansion of the attribute macro `apply` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
That's on 1.79.0